scopes/react/react/webpack/webpack.preview.config.ts

```diff
@@ -203,6 +203,7 @@ export default function (fileMapPath: string): Configuration {
         // https://www.smashingmagazine.com/2016/08/a-glimpse-into-the-future-with-react-native-for-web/
         // TODO: @uri please remember to remove after publishing evangelist and base-ui
         react: require.resolve('react'),
+        '@teambit/ui.mdx-scope-context': require.resolve('@teambit/ui.mdx-scope-context'),
```

Comment:
add `import '@teambit/ui.mdx-scope-context'` in the top, to make sure it will become a dep